dev/com.ibm.ws.security.jwt/src/com/ibm/ws/security/jwt/internal/JwtConfigUtil.java

```diff
@@ -53,4 +59,27 @@ public static String processProtectedString(Map<String, Object> props, String cf
         return secret;
     }
 
+    public static String getSignatureAlgorithm(Map<String, Object> props, String sigAlgAttrName) {
+        String defaultSignatureAlgorithm = "RS256";
+        String signatureAlgorithm = JwtUtils.trimIt((String) props.get(sigAlgAttrName));
+        boolean isBetaEnabled = ProductInfo.getBetaEdition();
+        if (!isBetaEnabled && isBetaAlgorithm(signatureAlgorithm)) {
+            if (tc.isDebugEnabled()) {
+                Tr.debug(tc, "A signature algorithm (" + signatureAlgorithm + ") is specified that's only available in the beta edition. Defaulting to " + defaultSignatureAlgorithm);
```

Comment:
If you do specify a beta Algorithm when not running a beta runtime, is it correct that this just defaults back without warning? You won't see any message unless debug trace is enabled. When not running in a beta, I think this should behave the same as if a completely unknown/invalid algorithm was specified ... in such a case, would there be an exception? An error or warning message? I don't think debug trace is enough.